packages/ui/src/components/va-alert/_variables.scss

```diff
@@ -1,42 +0,0 @@
-:root {
```

Comment:
Deletion of this file doesn't seem intended to me.

Reply:
😿 

![image](https://user-images.githubusercontent.com/5394573/135478418-eff0c6e2-4f7c-4378-9647-40425da98d29.png)